src/configuration-gui/Makefile.am

```diff
@@ -63,4 +68,4 @@ uidir = $(pkgdatadir)/ui
 EXTRA_DIST = abrt_gui.pc.in
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = abrt_gui.pc
```

Comment:
Why was this removed?

Reply:
This was some kind of strange git rebase error. I checked my local original patch this change was not there, but after rebasing the patch on top of current master the line was removed. Not sure why git did that, but it was not intended. Thanks for the catch.